src/Symfony/Bundle/FrameworkBundle/Client.php

```diff
@@ -41,7 +41,7 @@ public function __construct(KernelInterface $kernel, array $server = array(), Hi
     /**
      * Returns the container.
      *
-     * @return ContainerInterface
+     * @return ContainerInterface|null
```

Comment:
Can you add a comment above to specify when null can be returned? Something like `This method can return null when the Kernel has been shutdown or not started yet.`